packages/module/patternfly-docs/content/extensions/extended-components/examples/DetailsPage.md

```diff
@@ -0,0 +1,81 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: extensions
+subsection: react-component-groups
```

Comment:
https://github.com/patternfly/react-catalog-view/blob/f0c4194eb1231c894c693071079d33b0272ea360/packages/module/patternfly-docs/content/examples/CatalogItemHeader.md?plain=1#L4 suggests a subsection is possible.

Reply:
`react-component-groups` is the existing name of the package.  Will leave it as is for now.